src/Application.vala

```diff
@@ -46,16 +46,8 @@ public class Terminal.Application : Gtk.Application {
                 }
             }
 
-            // This is a hack to avoid using Gdk-Xii dependency.  Using present_with_time ()
-            // with the current event time does not work either on X11 or Wayland perhaps
-            // because the triggering event did not occur on the Terminal window?
-            // Using set_keep_above () at least works on X11 but not on Wayland
-            //TODO It may well be possible to use present () on Gtk4 so this needs revisiting
-            window_to_present.set_keep_above (true);
-            window_to_present.present ();
-            window_to_present.grab_focus ();
             Idle.add (() => {
```

Comment:
That's a wild hack, I'm glad we're removing it 😅 Does this still need to be wrapped in an idle?

Reply:
It seems to work OK on OS8 without the idle.  In `master` it doesn't work anyway on OS7 (and nor does this PR)